gradle.properties

```diff
@@ -29,6 +29,6 @@ VER_MOCKITO=3.3.3
 
 # Used for Maven Plugin
 VER_MAVEN_API=3.0
-VER_ECLIPSE_AETHER=1.1.0
+VER_ECLIPSE_AETHER=0.9.0.M2
```

Comment:
Makes sense to use the same Aether version as the minimum Maven version that's supported by the plugin 👍